Module/ifo/module.ifo.json

```diff
@@ -2121,6 +2121,13 @@
           "value": "space_derelict_k"
         }
       },
+      {
+        "__struct_id": 6,
+        "Area_Name": {
+          "type": "resref",
+          "value": "vis_cz_inte_ka1"
```

Comment:
This was a duplicate area I removed in the last update. This entry shouldn't be here.

Reply:
Weird that it's there - deleted